cirq/google/engine/client/quantum_v1alpha1/gapic/quantum_engine_service_client.py

```diff
@@ -74,13 +76,16 @@ def from_service_account_file(cls, filename, *args, **kwargs):
 
     from_service_account_json = from_service_account_file
 
-    def __init__(self,
-                 transport=None,
-                 channel=None,
-                 credentials=None,
-                 client_config=None,
-                 client_info=None,
-                 client_options=None):
+    def __init__(
+            self,
+            transport: Union[quantum_engine_service_grpc_transport.
+                             QuantumEngineServiceGrpcTransport] = None,
+            channel: grpc.Channel = None,
+            credentials: service_account.Credentials = None,
+            client_config: Dict = None,
```

Comment:
Is there really nothing we know about key and value type? Perhaps `Dict[str, Any]` applies?

Reply:
On the first pass I directly copied the types in the docstrings for each argument - I'll go back through now and narrow these down if possible.

Reply:
Done - it looks like these are all for protobufs in `Dict[str, Any]` format.